### PR TITLE
added an average rating score

### PR DESCRIPTION
**Pull request by Chepenik**
Small update also the favicon should be updated to a pepper :D

### CODE REVIEW

**Comment by Chepenik** [comment]
https://www.loom.com/share/93e8fa7ee46e4ad5b198ef78d6ecce5f

**Comment by RipNasty** [comment]
Did this with Conor.  Functionality works, hope it's all set!

File: client/src/components/SocialMediaShow.js [hunk 1]
```diff
@@ -58,26 +61,41 @@ const SocialMediaShow = (props) => {
...
+    )
+
+  const averagePepperRating =
+    socialMediaShow.comments.length > 0
+      ? (
+          socialMediaShow.comments.reduce(
+            (sum, comment) => sum + comment.rating,
+            0
+          ) / socialMediaShow.comments.length
+        ).toFixed(1)
```
Comment (dovakeen118) [inline client/src/components/SocialMediaShow.js:84]: data manipulation/ calculation should ideally occur on the back-end

**Review by dovakeen118 — COMMENTED** [review]
one small comment, then merge!

**Pull request merged**